x-pack/plugins/data_visualizer/public/application/common/components/expanded_row/geo_point_content_with_map/geo_point_content_with_map.tsx

```diff
@@ -64,7 +66,7 @@ export const GeoPointContentWithMap: FC<{
   return (
     <ExpandedRowContent dataTestSubj={'dataVisualizerIndexBasedMapContent'}>
       <DocumentStatsTable config={config} />
-      <ExamplesList examples={stats.examples} />
+      <ExamplesList examples={uniqueExamples} />
```

Comment:
Is this same component used in the file data visualizer? Wonder if duplicate examples are being removed there too?

Reply:
While this is not the same component as the file based one, luckily the file upload endpoint removes duplicates during the grokking process.